src/build/build.ts

```diff
@@ -59,10 +56,13 @@ export async function build(
     js: Object.assign({minify: true}, options.js),
   };
 
-  logger.info(`Preparing build...`);
-  await del([unbundledBuildDirectory, bundledBuildDirectory]);
+  if (options.insertDependencyLinks) {
+    logger.debug(
+        `Additional dependency links will be inserted into application`);
+  }
 
-  logger.info(`Building application...`);
+  logger.info(`Deleting build/ directory...`);
+  await del([buildDirectory]);
```

Comment:
Ehh, I've been burned enough by this to think it should be default. Any time any dependency path, source file path, dependency imports, or source imports change, you end up with ghost files in your build. It adds ~500ms on my machine, which is well worth it IMO.